src/NotificationInline/index.js

```diff
@@ -0,0 +1,202 @@
+      </View>
+   )
+
+   return (
+      <View
+         id={id}
+         display="block"
+         width="100%"
+         paddingHorizontal={paddingHorizontal}
+         paddingVertical={paddingVertical}
+         borderRadius={borderRadius}
+         backgroundColor={notificationBackgroundColor}
+         style={style}>
+         <View display="flex" flex="1 1 auto">
+            <View
+               flex="0 0 auto"
+               display="flex"
+               alignItems="center"
+               justifyContent="center"
+               float="left"
+               width={iconSize}
+               height={iconSize}
+               borderRadius="50%"
+               marginTop="3px"
+               marginRight={iconSpacing}>
+               <NotificationIcon
+                  width="100%"
+                  height="100%"
+                  color={notificationIconColor}
+               />
+            </View>
+            <View
+               flex="1 1 auto"
+               display="flex"
+               flexFlow="column[a-b] row[e-f]"
+               alignItems="flex-start[a-b] center[e-f]"
+               justifyContent="space-between">
+               <View paddingRight="12px">
+                  <Text
+                     size={textSize}
+                     weight="bold"
+                     color={notificationTextColor}>
+                     {title}
+                  </Text>
+                  <View display="block" width="100%" height="1px" />
+                  <Text
+                     size={textSize}
+                     lineHeight="140%"
+                     color={notificationTextColor}
+                     sizeMultiplier="0.95"
+                     style={{ opacity: 0.7 }}>
+                     {message}
+                  </Text>
+                  {children}
+                  {notificationButtons && buttonPlacement === 'bottom' && (
+                     <View marginTop={iconSpacing}>
+                        {notificationButtons}
+                     </View>
+                  )}
+               </View>
+               {notificationButtons && buttonPlacement === 'right' && (
+                  <View flex="0 0 auto">
+                     {notificationButtons}
+                  </View>
+               )}
+            </View>
+         </View>
+      </View>
+   )
+}
+
+NotificationInline.propTypes = {
+   backgroundColor: PropTypes.string,
+   borderRadius: PropTypes.string,
+   buttonPlacement: PropTypes.oneOf(['bottom', 'right']),
+   buttons: PropTypes.array,
+   children: PropTypes.node,
+   iconColor: PropTypes.string,
+   iconSpacing: PropTypes.string,
+   iconSize: PropTypes.string,
+   id: PropTypes.string,
+   message: PropTypes.string.isRequired,
+   paddingHorizontal: PropTypes.string,
+   paddingVertical: PropTypes.string,
+   style: PropTypes.object,
+   textColor: PropTypes.string,
+   textSize: PropTypes.string,
+   title: PropTypes.string.isRequired,
+   type: PropTypes.oneOf([
```

Comment:
How about keepin' it flex, like `type: PropTypes.oneOf(Object.keys(notificationTypes)).isRequired` ?